### ISSUE TITLE
idsly.bid [Anti-Adblock]

### ISSUE THREAD

**Issue by KiLL4FuNK** [uBlockOrigin/uAssets#1870]
URL(s) where the issue occurs

`https://idsly.bid/NLh51`

Describe the issue

Anti-Adblock

Versions

Browser/version: Chrome 65.0.3325.181
uBlock Origin version: 1.15.18

Settings

Default

**Comment by zaphodbeebblebrox** [comment]
`@@||idsly.bid^$generichide`

**Comment by okiehsch** [comment]
I am geo-blocked and if I access this site via proxy the captcha is broken, so fixing this is not straightforward on my end.
The site disables right click functionality and it also disables the shortcuts for opening the console or the source-code.
Does
````
idsly.bid##script:inject(abort-on-property-write.js, clickIE)
idsly.bid##script:inject(addEventListener-defuser.js, contextmenu, preventDefault)
idsly.bid##script:inject(addEventListener-defuser.js, keydown, disabledEvent)
idsly.bid##script:inject(nano-setInterval-booster.js)
idsly.bid##script:inject(setTimeout-defuser.js, getComputedStyle, 250)
````
fix all these issues without breaking anything?

**Comment by zaphodbeebblebrox** [comment]
Yes.
`idsly.bid##script:inject(addEventListener-defuser.js, keydown, disabledEvent)` does not appear to be needed. I can right click and solve the captcha without it.

**Comment by okiehsch** [comment]
Yes, I forgot them, if I add filters for a site I fix all ad-related issues, even if they can be fixed by "normal" filters.
Does
````
idsly.bid###link-view > a[href^="/go.php"]
idsly.bid##.col-md-3
````
work?

**Comment by zaphodbeebblebrox** [comment]
Bottom one works, top one doesn't.  For the top you could add `idsly.bid##img[src="https://idsly.bid/download3.png"]`.

**Comment by zaphodbeebblebrox** [comment]
Np. If you ever need someone with a U.S. IP, I'm willing to help.

**Comment by SMed79** [comment]
> I am geo-blocked

https://forums.lanik.us/viewtopic.php?f=88&t=37860
